packages/abi-gen/package.json

```diff
@@ -29,12 +31,19 @@
     "dependencies": {
         "@0xproject/typescript-typings": "^0.4.1",
         "@0xproject/utils": "^0.7.1",
-        "ethereum-types": "^0.0.1",
+        "@types/sleep": "^0.0.7",
+        "@types/tmp": "^0.0.33",
+        "chai": "^4.1.2",
         "chalk": "^2.3.0",
+        "dirty-chai": "^2.0.1",
```

Comment:
Ditto with `chai` and `dirty-chai`, testing deps should be listed under `devDependencies`. This is because end-users of this library do not need these deps to use the package. Listing under `devDependencies` means these deps don't get installed for end-users.

Reply:
Really appreciate your explanation around this.